packages/block-editor/src/components/block-preview/README.md

```diff
@@ -30,6 +30,8 @@ Width of the preview container in pixels. Controls at what size the blocks will
 
 `viewportWidth` can be used to simulate how blocks look on different device sizes or to make sure make sure multiple previews will be rendered with the same scale, regardless of their content.
 
+`viewportWidth` can be override in example object with the property `viewportWidth`.
```

Comment:
BlockPreview component can be used in very different contexts, I don't think we should be mentioning the "example" object here. Can we leave this out?

Reply:
Done

Reply:
Oh sorry, I was not clear enough, I was thinking we should remove the whole sentence from here.

Reply:
OK it's done